<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchAddDialog</class>
 <widget class="QDialog" name="BatchAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>文本：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Text"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>起始日期：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="start_y">
            <property name="minimum">
             <number>1900</number>
            </property>
            <property name="maximum">
             <number>2999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>年</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="start_m">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>月</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="start_d">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>31</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>日</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>终止日期：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="end_y">
            <property name="minimum">
             <number>1900</number>
            </property>
            <property name="maximum">
             <number>2999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>年</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="end_m">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>月</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="end_d">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>31</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>日</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>时间间隔：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="y_button">
            <property name="text">
             <string>年</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_button">
            <property name="text">
             <string>月</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="w_button">
            <property name="text">
             <string>周</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="d_button">
            <property name="text">
             <string>日</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_7">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BatchAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>BatchAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
